### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by ghost**
Some readme edits, making files consistent, adding a word, ensuring correct file extension in asset precomp example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Break this file up into two separate files:
 Here's `app/assets/javascripts/i18n/translations.js.i18njs`:
 
 ```yaml
----
+{}
```
Comment (camertron) [inline README.md:39]: Is this necessary? I thought the `---` yaml front matter was enough.

File: README.md [hunk 2]
```diff
@@ -68,17 +68,17 @@ locales:
 
 ### Asset Precompilation
 
-Any .i18njs file can be added to your application's asset precompile list just as you might do with any other type of asset. However, it can be inefficient to include the translations for every language in your Javascript bundle, especially if you have lots of translations. To avoid this, i18n-js-assets supports splitting translations up into individual files, one per locale, for precompilation. Try something like this in your config/application.rb:
+Any `.i18njs` file can be added to your application's asset precompile list just as you might do with any other type of asset. However, it can be inefficient to include the translations for every language in your Javascript bundle, especially if you have lots of translations. To avoid this, i18n-js-assets supports splitting translations up into individual files, one per locale, for precompilation. Try something like this in your `config/application.rb`:
 
 ```ruby
 config.assets.localized.add(
-  'i18n/translations.js',
+  'i18n/translations.js.i18njs',
```
Comment (camertron) [inline README.md:75]: I believe the asset pipeline will skip the asset if you add the `.i18njs` extension here.

**Review by camertron — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md:39]: Yeah I would have figured the same, but it returns `nil` when the file is read in. I had to give it the empty brackets `{}` before it read an empty hash out. Alternatively I could alter the [file reader](https://github.com/camertron/i18n-js-assets/blob/master/lib/i18n-js-assets/i18njs_file.rb#L9) to return an empty hash if it gets `nil` out of the file. What do you think?

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md:75]: It actually loaded the `i18n/translations.js` file provided by the `i18n-js` gem if I left the file extension off. I'd get sprockets directives in place of yaml when the file was read in :/

File: README.md (hunk 1 above)
Comment (camertron) [reply README.md:39]: I think changing the docs makes sense. I thought the empty `---` yaml front matter was something I used in the tests, but it looks like that's not the case.

File: README.md (hunk 2 above)
Comment (camertron) [reply README.md:75]: Huh that's curious... I wonder if that behavior changed in Rails 5 😢  So hard to keep up with this stuff.

**Pull request merged**